### DISCUSSION TITLE
Cant update my credentials.json for google play

### DISCUSSION THREAD

**Discussion by Kaddtechnologies** [codemagic-ci-cd/codemagic-docs#2463] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

When I go to any of my workflows that upload to google play, I cannot change nor update my credentials.json.
The option is not clickable and the field is disabled

### Expected results

I expect to be able to edit my credentials.json

### Actual results

I am locked out of that field therefore I cannot upload my app to google play

**Comment by icarusdust** [comment]
Hi @Kaddtechnologies, I would not suggest you edit your credentials, you can delete and re-create them with values you require. It is intentional that you are not allowed to edit secured values.

**Comment by icarusdust** [comment]
Closing this thread due to inactivity, let me know if you have any questions by replying to this thread or starting a new one.
